src/subsys/ngx_subsys_lua_util.c.tt2

```diff
@@ -839,8 +841,8 @@ ngx_[% subsys %]_lua_inject_ngx_api(lua_State *L, ngx_[% subsys %]_lua_main_conf
 
         if (rc != 0) {
             ngx_log_error(NGX_LOG_ERR, log, 0,
-                          "failed to load Lua code for the _G write guard: ",
```

Comment:
@agentzh The trailing comma here was a bug in all repositories (for both the load and run errors).